components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -398,7 +398,7 @@ def pickFirstSignature(condition, filterLambda):
                 return False
 
             # First check for null or undefined
-            pickFirstSignature("%s.isNullOrUndefined()" % distinguishingArg,
+            pickFirstSignature("%s.get().is_null_or_undefined()" % distinguishingArg,
```

Comment:
You found a part of codegen that was imported from gecko and has never been used in servo yet. Congratulations!